cmd/svcat/binding/unbind_cmd.go

```diff
@@ -84,28 +84,31 @@ func (c *unbindCmd) Run() error {
 }
 
 func (c *unbindCmd) deleteBinding() error {
-	err := c.App.DeleteBinding(c.Namespace, c.bindingName)
-	if err != nil {
-		return err
-	}
+	for _, bindingName := range c.bindingNames {
```

Comment:
There is existing logic to handle deleting multiple bindings, so we don't need to duplicate it here. Checkout https://github.com/kubernetes-incubator/service-catalog/blob/cd1c2fd0fe61f20196300fe004aedba9dc1eaa73/pkg/svcat/service-catalog/binding.go#L106 and I bet we can extract out that logic so that we can reuse it between unbind instance and deleting multiple bindings.

FYI, the goal is to not have any interesting non-cli logic in the cmd/svcat package. Anything that may be worth resusing should go into pkg/svcat. So whenever you are looking to add functionality to svcat, that's the place to put it.